awx/main/migrations/0064_v350_credential_plugins.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
+
+# AWX
+import awx.main.fields
+from awx.main.models import CredentialType
+
+
+def setup_tower_managed_defaults(apps, schema_editor):
+    CredentialType.setup_tower_managed_defaults()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('taggit', '0002_auto_20150616_2121'),
```

Comment:
Why would we add a 0002 migration dependency, as opposed to deleting the taggit app? And why do either of these things now?

Reply:
That's a great question.  If you can explain to me why Django automatically adds these when I run `awx-manage makemigrations`, points for you :)

Reply:
It looks like we already have the taggit 0002 migration as a dependency in our `0001_initial` migration.

I would think that our very own 0001 migration would be an ancestor of this migration, ergo, taggit 0002 is already an implicit dependency. It didn't do this for me when I added 1 simple field.

Also, just to re-state the general situation, I know of literally no reason to have the taggit app.

Reply:
well, it seems pretty clear that it did this for your migration because you have references to the auth app's User model and the taggit app's tag bonanza. For minor field changes, this is not relevant.

Maybe Django wants to add in redundant dependency links for all these cases, because they wouldn't do any harm.